docs/src/main/paradox/index.md

```diff
@@ -33,6 +33,6 @@ Various parts of Akka management can be used together for deploying Akka Cluster
   - [Akka Discovery Methods](discovery/index.md)
   - [Akka Cluster Management (HTTP)](cluster-http-management.md)
   - [Akka Cluster Management (JMX)](cluster-jmx-management.md)
-  - [Dynamic Log Levels](loglevels.md)
+  - [Dynamic Log Levels](loglevels/index.md)
```

Comment:
it would be nice if we could add a redirect from `loglevels.html` to `loglevels/index.html`, so we don't break existing links and lose SEO, but I guess it's not a big deal here.

Reply:
Should I open new pull request for it?

Reply:
I think it's fine like this TBH

Reply:
Ok, thanks.